File: 0027-remove-element/0027-remove-element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        if len(nums) == 0:
            return 0
        res_i = 0
        for check_i in range(0, len(nums)):
            if nums[check_i] != val:
                nums[res_i] = nums[check_i]
                res_i += 1
        return res_i

File: 0027-remove-element/0027-remove-element.java
class Solution {
    public int removeElement(int[] nums, int val) {
        if(nums.length == 0){
            return 0;
        }

        int res_i = 0;
        for(int check_i=0;check_i<nums.length;check_i++){
            if(nums[check_i] != val){
                nums[res_i] = nums[check_i];
                res_i++;
            }
        }

        return res_i;
    }
}